### PR TITLE
Move to 1.15

### PR DESCRIPTION
**Pull request by adrwa**
With 1.15 releasing soon it seems insignificant to focus on 1.14.

### CODE REVIEW

File: src/main/java/org/sandboxpowered/sandbox/fabric/SandboxResources.java [hunk 1]
```diff
@@ -57,12 +57,10 @@ protected boolean containsFile(String filename) {
     }
 
     @Override
-    public Collection<Identifier> findResources(ResourceType type, String path, int depth, Predicate<String> predicate) {
+    public Collection<Identifier> findResources(ResourceType type, String namespace, String path, int depth, Predicate<String> predicate) {
```
Comment (LemmaEOF) [inline src/main/java/org/sandboxpowered/sandbox/fabric/SandboxResources.java:60]: We should have a separate method for finding all resources at a certain path, regardless of namespace. That functionality is pretty essential for some of the stuff I enjoy doing.

File: src/main/java/org/sandboxpowered/sandbox/fabric/util/wrapper/RenderUtilImpl.java [hunk 2]
```diff
@@ -28,9 +28,9 @@ public void drawRepeating(int x, int y, int u, int v, int width, int height, flo
         GlStateManager.disableLighting();
         GlStateManager.disableFog();
         Tessellator tessellator_1 = Tessellator.getInstance();
-        BufferBuilder bufferBuilder_1 = tessellator_1.getBufferBuilder();
+        BufferBuilder bufferBuilder_1 = tessellator_1.getBuffer();
         GlStateManager.color4f(1.0F, 1.0F, 1.0F, 1.0F);
-        bufferBuilder_1.begin(7, VertexFormats.POSITION_UV_COLOR);
+        bufferBuilder_1.begin(7, VertexFormats.POSITION_COLOR_TEXTURE);
```
Comment (LemmaEOF) [inline src/main/java/org/sandboxpowered/sandbox/fabric/util/wrapper/RenderUtilImpl.java:33]: This looks like it's gonna swap the argument orders of color and texture. If there's a `POSITION_TEXTURE_COLOR` I think that's what we want for the vertex setups below.

**Review by LemmaEOF — APPROVED** [review]
Looking pretty good!

File: src/main/java/org/sandboxpowered/sandbox/fabric/SandboxResources.java (hunk 1 above)
Comment (adrwa) [reply src/main/java/org/sandboxpowered/sandbox/fabric/SandboxResources.java:60]: For our API we will, im not entirely sure why mojang made it this way, or maybe i completely misunderstood that param of the method (but don't think i did)

Comment (LemmaEOF) [reply src/main/java/org/sandboxpowered/sandbox/fabric/SandboxResources.java:60]: Ahh, this method is for inside a single resource/data pack. Looking at source, it looks like this method gets all the identifiers inside a certain folder of a certain namespace at a certain depth.

**Pull request merged**